srcpkgs/simulide/patches/00-glib.patch

```diff
@@ -0,0 +1,24 @@
+--- SimulIDE.pro 2020-08-13 22:39:49.000000000 +0300
++++ SimulIDE.pro 2020-10-22 03:08:27.819023767 +0300
+@@ -103,7 +103,9 @@
+     ../src/gpsim/*.h \
+     ../src/gpsim/devices/*.h \
+     ../src/gpsim/modules/*.h \
+-    ../src/gpsim/registers/*.h
++    ../src/gpsim/registers/*.h \
++     /usr/include/glib-2.0/glib.h \
++     /usr/lib/glib-2.0/include/glibconfig.h
+
+ INCLUDEPATH += ../src \
+     ../src/gui \
+@@ -140,7 +140,9 @@
+     ../src/gpsim \
+     ../src/gpsim/devices \
+     ../src/gpsim/modules \
+-    ../src/gpsim/registers
++    ../src/gpsim/registers \
++     /usr/include/glib-2.0 \
++     /usr/lib/glib-2.0/include
```

Comment:
I'm pretty sure this breaks cross compilation.

Reply:
I'd like to get some advise on handling this.

Reply:
I guess this chunk (in template file) would work (not tested):
```sh
CXXFLAGS="$($PKG_CONFIG --cflags glib-2.0)"
```

Reply:
With that change added, I don't think we need to patch.